### DISCUSSION TITLE
Seperate login page layout based on roles

### DISCUSSION THREAD

**Discussion by kanchansapkota27** [apache/superset#15167] [Ideas]
It would be better if users can set the landing page based on which role is logging in.
Suppose a public or gamma user not being able to see all the recent activities and charts along with saved queries if we only want them to see a list of dashboard.
While admin could see all those things.

**Comment by kamalkeshavani-aiinside** [comment]
I have requested a similar feature here:
https://github.com/apache/superset/issues/13356
You can upvote or add your requirements over it.
